### PR TITLE
T**DetId removed, TrackerTopology used

### PR DESCRIPTION
**Pull request by alesaggio**
Changes in Alignment/TrackerAlignment

### CODE REVIEW

**Review by pieterdavid — CHANGES_REQUESTED** [review]
Looks good, thanks!
General comment: do you want to merge this now, or add more similar changes first? (in the latter case, pleas add a "do not merge" or "work in progress" label)

File: Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc [hunk 1]
```diff
@@ -315,33 +315,35 @@ CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
...
 //////////////////////////////////////////////////////////////////////////////////////////////////
 //			Hit information in TID		                                	//
 /////////////////////////////////////////////////////////////////////////////////////////////////
           else if (SiStripDetId::TID == subdetId1)
           { 
              ++nhitinTID;
-             TIDDetId tidId1(detId1);
-             if (tidId1.isZMinusSide())
+	 
+			 //tTopo->...something...(...detId1...);
```
Comment (pieterdavid) [inline Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc]: I think these three lines (two comment, one with unused result) can be removed

File: Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc [hunk 2]
```diff
@@ -315,33 +315,35 @@ CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
...
+             //TIDDetId tidId1(detId1); //from previous code
+             tTopo->tidIsZMinusSide(detId1);
```
Comment (pieterdavid) [inline Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc]: I think these three lines (two comment, one with unused result) can be removed

Comment (pieterdavid) [reply Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc]: other question (github doesn't show me): did you manage to solve the space/tab issue?

Comment (alesaggio) [reply Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc]: Everything should be fine now ;)

Comment (pieterdavid) [reply Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc]: Could you push the commit that removes these lines then? ;-)

**Comment by alesaggio** [comment]
Let's wait until I have more files updated, I created the label

File: Alignment/TrackerAlignment/BuildFile.xml [hunk 3]
```diff
@@ -15,4 +15,5 @@
 <use   name="Geometry/TrackerGeometryBuilder"/>
 <export>
   <lib   name="1"/>
+  <use   name="DataFormats/TrackerCommon"/>
```
Comment (pieterdavid) [inline Alignment/TrackerAlignment/BuildFile.xml]: Sorry for being picky, I don't think we need this one

**Review by pieterdavid — APPROVED** [review]

**Pull request merged**